### PR TITLE
Rework GCP Audit and DNS logs sections

### LINKED ISSUE #4672: Rework GCP integration's Audit logs and DNS queries sections
While working on #4644, we noticed that the documentation regarding the ingestion of Google Audit logs and DNS logs is rather incomplete. It only shows some screenshots of how the ingested logs would look in the Kibana WUI, but it doesn't explain to the users how to configure their GCP services in order to be able to pull those logs with Wazuh.

The sections that should be reworked are the following:
- https://documentation.wazuh.com/current/gcp/supported-services/index.html#audited-resources
- https://documentation.wazuh.com/current/gcp/supported-services/index.html#dns-queries 

The Kibana screenshots of those sections should be updated too, removing the hightliting and blurred text and replacing the red rectangles with new ones using the Wazuh brand color: #00A9E5.

### PR DESCRIPTION
**Pull request by noise-kngdm**
|Related issue|
|---|
|Closes #4672|

## Description
In this PR we modify the documentation regarding collecting GCP [audit logs](https://documentation.wazuh.com/current/gcp/supported-services/index.html#audited-resources) and [DNS logs](https://documentation.wazuh.com/current/gcp/supported-services/index.html#dns-queries), which was rather uncomplete.

## Checks
- [x] It compiles without warnings.
- [x] Spelling and grammar. 
- [x] Used impersonal speech. 
- [x] Used uppercase only on nouns. 
- [x] Updated the `redirect.js` script if necessary (check [this guide](https://github.com/wazuh/wazuh-documentation/blob/master/NEW_RELEASE.md)).

### CODE REVIEW

File: source/gcp/supported-services/cloud_audit_logs.rst [hunk 1]
```diff
@@ -7,34 +7,76 @@
 Audited resources
 =================
 
-Google Cloud maintains three `audit logs <https://cloud.google.com/monitoring/api/resources#tag_audited_resource>`__ for each Google Cloud project, folder, and organization: **Admin Activity**, **Data Access**, and **System Event**. Wazuh supports collecting those logs by using the **gcp-pubsub** module. Details on how to configure the module can be found :ref:`gcp-pubsub configuration reference <gcp-pubsub>`.
+Google audit logs
+-----------------
 
-.. thumbnail:: ../../images/gcp/gcp-overview.png
+Google Cloud maintains four `audit logs <https://cloud.google.com/logging/docs/audit>`__ for each Google Cloud project, folder, and organization: **Admin Activity**, **Data Access**, **System Event**, and **Policy Denied**.
```
Comment (CarlosRS9) [inline source/gcp/supported-services/cloud_audit_logs.rst:13]:
```suggestion
Google Cloud provides four types of `audit logs <https://cloud.google.com/logging/docs/audit>`__ for each Google Cloud project, folder, and organization:
```

Comment (CarlosRS9) [reply source/gcp/supported-services/cloud_audit_logs.rst:13]: it is redundant to list the log types and then to list them again providing more detail

File: source/gcp/supported-services/cloud_audit_logs.rst [hunk 2]
```diff
@@ -7,34 +7,76 @@
...
+
+* **Admin Activity** audit logs contain log entries for API calls or other administrative actions that modify the configuration or metadata of resources.
+
+* **Data Access** audit logs contain API calls that read the configuration or metadata of resources, as well as user-driven API calls that create, modify or read user-provided resource data.
+
+* **System Event** audit logs contain log entries for Google Cloud administrative actions that modify the configuration of resources. These audit logs are generated by the Google system, therefore, no direct user action will drive them.
+
+* **Policy Denied** audit logs are recorded when a Google Cloud service denies access to a user or service account because of a security policy violation.
+
+Wazuh is able to collect and analyze those logs using the :ref:`GCP Pub/Sub integration <pubsub>`.
```
Comment (CarlosRS9) [inline source/gcp/supported-services/cloud_audit_logs.rst:23]:
```suggestion
Wazuh is able to collect and analyze these log types using the :ref:`GCP Pub/Sub integration <pubsub>`.
```

File: source/gcp/supported-services/dns_queries.rst [hunk 3]
```diff
@@ -7,8 +7,67 @@
...
+    :align: center
+    :width: 100%
+
+After selecting the ``Exists in`` button, only Google Cloud-related events will appear in the Kibana dashboard.
+
+.. thumbnail:: ../../images/gcp/gcp-kibana-dns-filtered-logs.png
+    :align: center
+    :width: 100%
+
+If you need information about all the Google services with audit logs that the Wazuh GCP module could analyze, visit `the Google Cloud documentation <https://cloud.google.com/logging/docs/audit/services>`__.
```
Comment (CarlosRS9) [inline source/gcp/supported-services/dns_queries.rst:73]:
```suggestion
Check the `the Google Cloud documentation <https://cloud.google.com/logging/docs/audit/services>`_ to learn more about the different Google services capable of writing audit logs.
```
Use impersonal if possible

**Review by CarlosRS9 — CHANGES_REQUESTED** [review]

File: source/gcp/supported-services/cloud_audit_logs.rst [hunk 4]
```diff
@@ -7,34 +7,76 @@
...
     :width: 100%
 
-* **Data Access** audit logs contain API calls that read the configuration or metadata of resources, as well as user-driven API calls that create, modify or read user-provided resource data.
+Finally, use the following query to collect all the audit logs from every project. It is possible to edit it to only collect audit logs from a particular project.
+
+.. code-block:: none
+
+    logName=~("projects/.*/logs/cloudaudit.googleapis.com%2F(activity|data_access|system_event|policy)")
+
+If it is not necessary to filter any logs out of the sink, click on the ``CREATE SINK`` button to create it.
```
Comment (CarlosRS9) [inline source/gcp/supported-services/cloud_audit_logs.rst:52]:
```suggestion
#. If it is not necessary to filter any logs out of the sink, click on the ``CREATE SINK`` button to create it.
```

File: source/gcp/supported-services/cloud_audit_logs.rst [hunk 5]
```diff
@@ -7,34 +7,76 @@
...
+Kibana dashboard visualization
+------------------------------
+
+After configuring the GCP Pub/Sub module to fetch the audit logs from Google Cloud, it is possible to visualize the alerts generated in the Kibana dashboard.
+
+.. thumbnail:: ../../images/gcp/gcp-overview.png
+    :align: center
+    :width: 100%
+
+Google Cloud logs can be filtered on Kibana by the ``data.gcp.logName`` field:
```
Comment (CarlosRS9) [inline source/gcp/supported-services/cloud_audit_logs.rst:70]:
```suggestion
#. Google Cloud logs can be filtered on Kibana by the ``data.gcp.logName`` field:
```

File: source/gcp/supported-services/cloud_audit_logs.rst [hunk 6]
```diff
@@ -7,34 +7,76 @@
...
 
-.. thumbnail:: ../../images/gcp/gcp-data-access.png
+.. thumbnail:: ../../images/gcp/gcp-kibana-log-filter.png
     :align: center
     :width: 100%
 
-* **System Event** audit logs contain log entries for Google Cloud administrative actions that modify the configuration of resources. These audit logs are generated by the Google system. Therefore, no direct user action will drive them.
+After selecting the ``Exists in`` button, only Google Cloud-related events will appear in the Kibana dashboard.
```
Comment (CarlosRS9) [inline source/gcp/supported-services/cloud_audit_logs.rst:76]:
```suggestion
#. After selecting the ``Exists in`` button, only Google Cloud-related events will appear in the Kibana dashboard.
```

File: source/gcp/supported-services/cloud_audit_logs.rst [hunk 7]
```diff
@@ -7,34 +7,76 @@
...
 
-.. thumbnail:: ../../images/gcp/gcp-system-event.png
+.. thumbnail:: ../../images/gcp/gcp-kibana-filtered-logs.png
     :align: center
     :width: 100%
 
-If you want to take a look at all the supported Google services with audit logs by the Wazuh GCP module, check `this link <https://cloud.google.com/logging/docs/audit/services>`__. 
+If you need information about all the Google services with audit logs that the Wazuh GCP module could analyze, visit `the Google Cloud documentation <https://cloud.google.com/logging/docs/audit/services>`__.
```
Comment (CarlosRS9) [inline source/gcp/supported-services/cloud_audit_logs.rst:82]:
```suggestion
Visit the `the Google Cloud documentation <https://cloud.google.com/logging/docs/audit/services>`__ to learn more about the different Google services capable of writing audit logs.
```

**Review by CarlosRS9 — CHANGES_REQUESTED** [review]

File: source/gcp/supported-services/cloud_audit_logs.rst (hunk 5 above)
Comment (noise-kngdm) [reply source/gcp/supported-services/cloud_audit_logs.rst:72]: I think the kibana section looks better without having tasks with only two items.

File: source/gcp/supported-services/dns_queries.rst [hunk 8]
```diff
@@ -7,8 +7,67 @@
...
-Wazuh has default rules for `DNS queries <https://cloud.google.com/monitoring/api/resources#tag_dns_query>`__ to a private DNS handled by the `Google Cloud DNS <https://cloud.google.com/dns/docs>`__ service. Those logs can be collected using the **gcp-pubsub** module. Details on how to configure the module can be found :ref:`gcp-pubsub configuration reference <gcp-pubsub>`.
+Wazuh has default rules for DNS queries made to a private DNS handled by the `Google Cloud DNS <https://cloud.google.com/dns/docs>`__ service. Those logs can be collected using the **gcp-pubsub** module. 
 
-.. thumbnail:: ../../images/gcp/gcp-dns-query.png
+Configure Google DNS logs collection
+------------------------------------
+
+To collect the DNS queries made to the Google DNS service it is necessary to enable DNS logging. To do that, follow the `Google Cloud DNS logging guide <https://cloud.google.com/dns/docs/monitoring>`_.
+
+Once DNS Cloud logging is configured, the generated logs must be ingested into a Pub/Sub topic so that Wazuh is able to collect them using the `Pub/Sub integration <pubsub>`. To achieve that, it is necessary to define a custom log router.
```
[the remark this answers is not in the document]
Comment (noise-kngdm) [reply source/gcp/supported-services/dns_queries.rst]: Fixed in [766431d](https://github.com/wazuh/wazuh-documentation/pull/4793/commits/766431db2d226ce17497762321c172d1e87fd3fd).

File: source/gcp/supported-services/dns_queries.rst (hunk 3 above)
Comment (noise-kngdm) [reply source/gcp/supported-services/dns_queries.rst]: The paragraph was included by mistake. I removed it in [766431d](https://github.com/wazuh/wazuh-documentation/pull/4793/commits/766431db2d226ce17497762321c172d1e87fd3fd).

**Review by CarlosRS9 — COMMENTED** [review]
Please review and fix the following warning which appears when attempting to compile the documentation:

**Review by CarlosRS9 — CHANGES_REQUESTED** [review]
See the previous message

**Comment by noise-kngdm** [comment]
Fixed it on 94b3b13, now it compiles without warnings:
image

**Review by CarlosRS9 — APPROVED** [review]
LGTM!

**Review by s-ocando — APPROVED** [review]

**Pull request merged**